source/common/config/delta_subscription_impl.h

```diff
@@ -118,13 +115,22 @@ class DeltaSubscriptionImpl
     }
   }
 
+  envoy::api::v2::DeltaDiscoveryRequest stateOfRequest() const { return request_; }
```

Comment:
I think either call this `internalRequestStateForTest()` or create a friended testing peer would be ideal. Or even having the `DeltaSubscriptionImpl` somehow emit an initial request into the mocks and then validate.

Reply:
Done.